06_gpu_and_ml/protein-folding/boltz_predict.py

```diff
@@ -121,20 +130,21 @@ def main(
     timeout=10 * MINUTES,
     gpu="H100",
 )
-def boltz_inference(boltz_input_yaml: str, args="") -> bytes:
+def boltz_inference(boltz_input_yaml: str, input_msa: str, args: str = "") -> bytes:
     import shlex
     import subprocess
 
     input_path = Path("input.yaml")
     input_path.write_text(boltz_input_yaml)
 
+    msa_path = Path("seq1.a3m")
```

Comment:
This will only work if this filename is hard-coded in the YAML file.

That means it will fail if the input YAML has more than one protein in it, e.g. when trying to do structure prediction for interacting proteins. That's a pretty big use case for Boltz.

We could get around this by mounting a local directory so that the YAML file and any associated files are all pulled in.